src/main/java/models/ProductModel.java

```diff
@@ -13,11 +13,12 @@
 @Builder
 @AllArgsConstructor
 public class ProductModel {
-    public final String name;
-    public final Currency priceCurrency;
-    public final BigDecimal price;
-    public final Boolean isOnSale;
-    public final BigDecimal priceBeforeSale;
-    public final Boolean isOutOfStock;
-    public final String imageUrl;
+	public final String name;
+	public final String model;
+	public final Currency priceCurrency;
+	public final BigDecimal price;
+	public final Boolean isOnSale;
+	public final BigDecimal priceBeforeSale;
+	public final Boolean isOutOfStock;
+	public final String imageUrl;
```

Comment:
Another set of redundant indentations I think?

Reply:
Formatting corrected, thanks